code/__DEFINES/~skyrat_defines/reagents.dm

```diff
@@ -0,0 +1,2 @@
+/// This reagent is useful for blood regeneration. Useful for Hemophages.
+#define REAGENT_BLOOD_REGENERATING (1<<9)
```

Comment:
this is totally gonna conflict, do you think you could move this to a `chemical_flags_skyrat` var, like was done for the forging trait?